utils/print_qor.py

```diff
@@ -7,17 +7,16 @@
 import subprocess
 
 
-def scan_runtime(fname):
+def scan_runtime(step, fname):
```

Comment:
Can you also output the overhead for each step.  So basically output step runtime and overhead runtime.  Make sense?

Reply:
Sure, makes sense. Done